content/en/user-guide/integrations/openshift/index.md

```diff
@@ -0,0 +1,68 @@
+
+OpenShift Web Console will automatically detect the Devfile and display the import strategy. A unique application name will be generated to the application grouping to label your resources. A unique name will also be provided to the component that will be used to name associated resources. You can edit these values if you want.
+
+Click on **Create** to deploy LocalStack on OpenShift.
+
+### Viewing the LocalStack deployment
+
+You can see the build status of the LocalStack deployment in the **Topology** view.
+
+<img src="openshift-topology-view.png" alt="OpenShift Topology view" title="OpenShift Topology view" width="900" />
+<br><br>
+
+After successful deployment, you can see the **localstack-dev-spaces** pod in the **Topology** view. Click on the pod to view the details. You will be able to see the following details:
+
+- Running pods along with the status and logs.
+- Builds for your existing pods and an option to create new builds.
+- Exposed services along with the service port and the pod port.
+- Exposed routes for your deployed pods on the cluster.
+
+<img src="localstack-dev-spaces.png" alt="LocalStack Dev Spaces" title="LocalStack Dev Spaces" width="900" />
+<br><br>
+
+### Creating AWS resources on OpenShift
+
+Click on the **localstack-dev-spaces** pod to view the details. You will be able to see the exposed route for the LocalStack container. Copy the route URL and use it to interact with the LocalStack container.
+
+To create an S3 bucket and an SQS queue, run the following commands:
+
+{{< command >}}
+$ export AWS_ENDPOINT_URL=<localstack-route-url>
+$ awslocal s3 mb s3://my-bucket
+$ awslocal sqs create-queue --queue-name my-queue
+{{< /command >}}
+
+In the above commands, replace `<localstack-route-url>` with the route URL of the LocalStack container. The `AWS_ENDPOINT_URL` environment variable is used to specify the endpoint URL of the LocalStack container. 
+
+{{< alert title="Note" >}}
+By default, the endpoint URL for `awslocal` is `http://localhost:4566`. Since we are running LocalStack on OpenShift, we need to specify the route URL of the LocalStack container. You can swap `awslocal` with the AWS CLI, by specifying the additional `--endpoint-url` parameter.
+{{< /alert >}}
+
+You can further use integrations, such as [CDK](https://docs.localstack.cloud/user-guide/integrations/aws-cdk/), [SAM CLI](https://docs.localstack.cloud/user-guide/integrations/aws-sam/), and [Terraform](https://docs.localstack.cloud/user-guide/integrations/terraform/), to interact with the Ephemeral Instance. In these integrations, you can change the `LOCALSTACK_HOSTNAME` environment variable to the endpoint URL of the Ephemeral Instance.
```

Comment:
The `LOCALSTACK_HOSTNAME` config is deprecated and should no longer being used. We've standardized around `AWS_ENDPOINT_URL`, which is supported by most of our tools these days - let's better reference that here..